streams/src/main/java/org/apache/kafka/streams/processor/StateRestoreListener.java

```diff
@@ -37,6 +37,9 @@
  * These two interfaces serve different restoration purposes and users should not try to implement both of them in a single
  * class during state store registration.
  *
+ * Also note that standby tasks restoration process are not monitored via this interface, since a standby task keep
```

Comment:
not: missing `<p>` between the paragraphs. -- Seems they are also missing on other places in this file -- can we fix all of them?

Reply:
> standby tasks restoration process

Is "restoration" the best wording?

Reply:
Ack.